src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.cs

```diff
@@ -1671,7 +1671,7 @@ internal CalendarId[] CalendarIds
         {
             get
             {
-                if (_waCalendars == null)
+                if (_waCalendars == null && !GlobalizationMode.Invariant)
```

Comment:
Is this needed? based on this https://github.com/dotnet/runtime/pull/37527/files/48393f74f45999eb53ce79c7cfda61d7c30fb691..eee299b30be940db2a9857a70aab76de6a5eb55d#diff-235473f71512930b24ea62cbe21024f6R630 `_waCalendars` shouldn't be null for `Invariant` right?

Reply:
Yes but illinker cannot statically detect that

Reply:
Ah I see, this is just to help the linker trim this code... ok cool.